common/helpers/move/get-locals.test.js

```diff
@@ -296,6 +296,38 @@ describe('Move helpers', function () {
       })
     })
 
+    context('when move doesn’t have a profile', function () {
+      let params
+
+      beforeEach(function () {
+        req.move = {
+          id: 'moveId',
+          status: 'requested',
```

Comment:
Can we make this test for `profile: null`. I don't think this will prevent the error we've seen because of this:

>Default values in destructuring assignement only work if the variables either don't exist or their value is set to undefined . Any other value, including null , false and 0 , bypasses the default values in the destructuring statement. You can combine default values with renaming in the destructuring assignment.

I think it will still come through as `null` instead of an empty object and cause the error.